app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
You can leave only Player here
(abstract class must have at least one abstract method, class Elf - not)

Reply:
I like that styling, since our Elf is pretty abstract class and you explicitly mention this )